db/index.ts

```diff
@@ -0,0 +1,62 @@
+interface BaseEvent {
+  externalId: string;
+  category: [string];
+  title: string;
+  city: string;
```

Comment:
Could this be a `location` instead of a `city`? This naming would fit better for online events.